.gitlab-ci.yml

```diff
@@ -1,21 +1,44 @@
-
 include:
-  - project: 'Northern.tech/Mender/mendertesting'
-    file: '.gitlab-ci-github-status-updates.yml'
+  - project: "Northern.tech/Mender/mendertesting"
+    file: ".gitlab-ci-github-status-updates.yml"
   - local: .gitlab-ci-staging-tests.yml
     rules:
       - if: $RUN_TESTS_STAGING == "true"
   - local: .gitlab-ci-default-pipeline.yml
     rules:
-      - if: $RUN_TESTS_STAGING != "true"
+      - if: $RUN_TESTS_STAGING == "true"
+        when: never
+      - if: $RUN_FULL_INTEGRATION == "true"
+        when: never
+  - local: .gitlab-ci-full-integration.yml
+    rules:
+      - if: $RUN_TESTS_FULL_INTEGRATION == "true"
 
 stages:
   - test
   - build
   - publish
 
 variables:
-  RUN_TESTS_STAGING: "false"
+  RUN_TESTS_STAGING:
+    description: "Run staging integration tests"
+    value: "false"
+  RUN_TESTS_FULL_INTEGRATION:
+    description: |
+      Run full integration tests.
+      NOTE: Can only run on protected branches
+    value: "false"
+    options:
+      - "false"
+      - "true"
+
+  DOCKER_VERSION:
+    description: "Docker version to use in jobs"
+    value: "27"
```

Comment:
Having it explicit in the `image:` tag will make it more visible and, hopefully, we will update it more often.

Reply:
In my experience the fields defined at the job level tends to be forgotten and get scattered around in different files. By centralizing the versions to a single variable I was hoping to cope with this issue.

Reply:
We'll try it out :wink: